src/pages/edit-deck-page/edit-deck-page.tsx

```diff
@@ -46,7 +46,7 @@ export const EditDeckPage = () => {
 
   async function fetchDeckAndRefresh() {
     if (deckId === undefined) {
-      throw new Error('deckId cannot be undefined'); // Todo: maybe route to a 404 page??
```

Comment:
did we want to start routing to this new page?

Reply:
It automatically routes it to the page. I just kept it because the type checker was complaining. But I guess we could explicitly send the user to /404 or something.

Reply:
ah I see! bc `/edit-deck` on its own is an invalid route

we'll probs do some better error checking in the future since our endpoints can fail
i think i'll help/work on improving that post demo though